### PR TITLE
Added initial front-end part for user registration

### PR DESCRIPTION
**Pull request by orestch**
Please, navigate to http://host:port/accounts/ (http://localhost:8000/accounts/ by default) to check user registration

### CODE REVIEW

**Comment by hutsulyk** [comment]
Remove servers, services, checks tabs when sign in/up

File: moninag/registration/views.py [hunk 1]
```diff
@@ -26,18 +28,32 @@ def activate(request, activation_key):
...
 def auth_view(request):
-    username = request.POST.get('username', '')
-    password = request.POST.get('password', '')
-    user = auth.authenticate(username=username, password=password)
-
-    if user is not None:
-        if user.is_active:
-            auth.login(request, user)
-            return HttpResponseRedirect('/accounts/profile')
+    json = {
```
Comment (lhalam) [inline moninag/registration/views.py]: deleted object json, no need to form and use it

File: moninag/registration/views.py [hunk 2]
```diff
@@ -26,18 +28,32 @@ def activate(request, activation_key):
...
+
+    data = loads(request.body.decode('utf-8'))
+
+
+    if request.method == 'POST':
+        email = data.get('email').strip().lower()
+        password = data.get('password').strip()
+
+        user = auth.authenticate(username=email, password=password)
+        if user is not None:
```
Comment (lhalam) [inline moninag/registration/views.py]: transform  to `if user:`

File: moninag/registration/views.py [hunk 3]
```diff
@@ -26,18 +28,32 @@ def activate(request, activation_key):
 
 
 def auth_view(request):
```
Comment (lhalam) [inline moninag/registration/views.py]: rename this function

**Comment by hutsulyk** [comment]
Add to `.gitignore` path to all js files in static

**Comment by hutsulyk** [comment]
Remove `auth_app.js` file from commit :slightly_smiling_face:

File: moninag/static/src/ts/auth/auth.module.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        FormsModule,
+        HttpModule,
+        MaterialModule.forRoot(),
+        RouterModule.forRoot([
+            {
+                path: '',
+                redirectTo: '/accounts',
+                pathMatch: 'full'
+            },
+
```
Comment (lhalam) [inline moninag/static/src/ts/auth/auth.module.ts]: delete empty lines

File: moninag/registration/urls.py [hunk 5]
```diff
@@ -1,20 +1,20 @@
...
 
 # Uncomment the next two lines to enable the admin:
 # from django.contrib import admin
 # admin.autodiscover()
 
 urlpatterns = [
     # auth
-    url(r'^$', login, name='login'),
-    url(r'^auth/$', auth_view, name='auth_view'),
+    url(r'^$', auth, name='auth'),
```
Comment (lhalam) [inline moninag/registration/urls.py:10]: updat url_path to
auth/logon
auth/register
auth/activate
auth/logout
api/1/profile
api/1/profile/inactive

add functionality auth/recovery 
remove register_success

**Review by lhalam — APPROVED** [review]

**Pull request merged**